cmd/debos/debos.go

```diff
@@ -280,6 +285,11 @@ func main() {
 				exitcode = 1
 				return
 			}
+
+			scratchsizeMB := int(size / 1000 / 1000)
+			if scratchsizeMB < 2048 {
+				log.Printf("WARNING: Scratch size of %dMB is less than recommended 2048MB\n", scratchsizeMB)
```

Comment:
fwiw scratchsize is in particular surprising when it's e.g. less then the vm's memory as that makes it somewhat pointless.  2G is quite a bit for most systems we tend to be build, for most 1G or even 512M would still be fine.

Reply:
changed it to say "recommended minimum" and set that limit to be 512M

Reply:
I could check if its smaller than RAM here, but i don't think it's worth it.